data/Witkowski_1991/metadata.yml

```diff
@@ -70,8 +70,8 @@ contexts:
   category: entity_context
   var_in: plant_age
   values:
-  - value: 10
-  - value: 21
+  - value: ten
```

Comment:
I wonder if it's an issue with converting to character again. Did you try '10' and '21'?

Reply:
Yes, I tried '10' and '21'; I checked `metadata_check_custom_R_code("Witkowski_1991)` to see how it was read in; I tried retyping the cells; I tried assigning it to a different context category; I tried adding "mutate to as.character" through custom_R_code and nothing worked. All rows in the column have a value, so it isn't being read in strangely. I'd much rather not use this trick, but also needed to move on. @yangsophieee

Reply:
That's so weird!!! And yes I think we should move on.